man/sd_bus_close.xml

```diff
@@ -59,6 +65,15 @@
     bus object so that it may be freed. Since these three operations are frequently done together a helper call
     <citerefentry><refentrytitle>sd_bus_flush_close_unref</refentrytitle><manvolnum>3</manvolnum></citerefentry> is
     provided that combines them into one.</para>
+
+    <para><function>sd_bus_default_flush_close()</function> is similar to
+    <function>sd_bus_flush_close_unref</function>, but does not take a bus pointer argument and instead
+    iterates over any of the "default" busses opened by
+    <citerefentry><refentrytitle>sd_bus_default</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
+    <citerefentry><refentrytitle>sd_bus_default_user</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
+    <citerefentry><refentrytitle>sd_bus_default_system</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
+    and similar calls. <function>sd_bus_default_flush_close()</function> call is particularly useful to clean
```

Comment:
`<function>sd_bus_default_flush_close()</function> call` => `<function>sd_bus_default_flush_close()</function>` or `The <function>sd_bus_default_flush_close()</function> call`